LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2Ability_WeaponAbilities.uc

```diff
@@ -0,0 +1,123 @@
+//---------------------------------------------------------------------------------------
+//  FILE:    X2Ability_SMGAbilities.uc
+//  AUTHOR:  Amineri (Long War Studios)
+//  PURPOSE: Adds stat modifier abilities for all tech tiers of SMG

```

Comment:
I would update the text here if you plan to use it :smile:

Reply:
D'oh yeah I meant to give him credit for that and put myself as the author :P. I'll make sure to get that taken care of.

Reply:
Fixed. Now the line reads me as the author and gives credit to Amineri.